dhctl/pkg/kubernetes/actions/manifests/manifests.go

```diff
@@ -69,189 +73,196 @@ func GetDeckhouseDeployTime(deployment *appsv1.Deployment) time.Time {
-          value: "6443"
-        - name: LOG_LEVEL
-          value: PLACEHOLDER
-        - name: DECKHOUSE_BUNDLE
-          value: PLACEHOLDER
-        - name: DECKHOUSE_POD
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: HELM_HOST
-          value: "127.0.0.1:44434"
-        - name: ADDON_OPERATOR_CONFIG_MAP
-          value: deckhouse
-        - name: ADDON_OPERATOR_PROMETHEUS_METRICS_PREFIX
-          value: deckhouse_
-        - name: ADDON_OPERATOR_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: ADDON_OPERATOR_LISTEN_ADDRESS
-          valueFrom:
-            fieldRef:
-              fieldPath: status.podIP
-        - name: HELM3LIB
-          value: "yes"
-        - name: KUBERNETES_DEPLOYED
-          value: PLACEHOLDER
-        volumeMounts:
-        - mountPath: /tmp
-          name: tmp
-        - mountPath: /.kube
-          name: kube
-        ports:
-        - containerPort: 9650
-          name: self
-        - containerPort: 9651
-          name: custom
-        readinessProbe:
-          httpGet:
-            path: /ready
-            port: 9650
-          initialDelaySeconds: 5
-          # fail after 10 minutes
-          periodSeconds: 5
-          failureThreshold: 120
-        workingDir: /deckhouse
-      hostNetwork: true
-      dnsPolicy: Default
-      serviceAccountName: deckhouse
-      nodeSelector:
-        node-role.kubernetes.io/master: ""
-      tolerations:
-      - operator: Exists
-      volumes:
-      - emptyDir:
-          medium: Memory
-        name: tmp
-      - emptyDir:
-          medium: Memory
-        name: kube
-`
-
-	var deployment appsv1.Deployment
-	err := yaml.Unmarshal([]byte(deckhouseDeployment), &deployment)
-	if err != nil {
-		panic(err)
+	deckhouseDeployment := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "deckhouse",
+			Namespace: "d8-system",
+			Labels: map[string]string{
+				"heritage":                     "deckhouse",
+				"app.kubernetes.io/managed-by": "Helm",
+			},
+			Annotations: map[string]string{
+				"meta.helm.sh/release-name":      "deckhouse",
+				"meta.helm.sh/release-namespace": "d8-system",
+			},
+		},
+		Spec: appsv1.DeploymentSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": "deckhouse",
+				},
+			},
+			Strategy: appsv1.DeploymentStrategy{
+				Type: appsv1.RecreateDeploymentStrategyType,
+			},
+		},
 	}
 
-	return ParametrizeDeckhouseDeployment(&deployment, params)
+	deckhousePodTemplate := apiv1.PodTemplateSpec{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: deckhouseDeployment.Spec.Selector.MatchLabels,
+		},
+		Spec: apiv1.PodSpec{
+			HostNetwork:        true,
+			DNSPolicy:          apiv1.DNSDefault,
+			ServiceAccountName: "deckhouse",
+			Tolerations: []apiv1.Toleration{
+				{Operator: apiv1.TolerationOpExists},
+			},
+			Volumes: []apiv1.Volume{
```

Comment:
I see volumes for pod, but i do not find Volume mounts https://github.com/deckhouse/deckhouse/pull/716/files#diff-49aea6002f1b4637e82d319b8c945b0eacbaee54c97975123db358619c35cca3L213

It need to prevent restarts Deckhouse after restart bootstrap.